saleor/graphql/views.py

```diff
@@ -162,7 +175,9 @@ def _handle_query(self, request: HttpRequest) -> JsonResponse:
         return JsonResponse(data=result, status=status_code, safe=False)
 
     def handle_query(self, request: HttpRequest) -> JsonResponse:
-        with tracer.start_as_current_span("http", kind=trace.SpanKind.SERVER) as span:
+        with tracer.start_as_current_span(
+            "http", service_scope=True, kind=SpanKind.SERVER
```

Comment:
Are we sure all data in `http` should be exposed to `SERVICE` tracker?

Reply:
Yes, we agreed that all the attributes are suitable for the `service` scope